Examples/Deployment/README.md

```diff
@@ -13,6 +13,13 @@ cd swift-aws-lambda-runtime/Examples/Deployment
 
 Note: The example scripts assume you have [jq](https://stedolan.github.io/jq/download/) command line tool installed.
 
```

Comment:
This information seems to apply to all types of deployments here (CLI, SAM, Serverless) so I made a special section for it. For an M1 user, this information is really critical.